tools/autograd/templates/VariableType.cpp

```diff
@@ -1,3 +1,4 @@
+#include "Python.h"
```

Comment:
Why does this need Python.h?

Reply:
For warning squashing. But this is obsoleted by #3187 so I'll delete it.